Tests/Translator/Extractor/JMSTranslatorBundle/AdminExtractorTest.php

```diff
@@ -77,8 +77,21 @@ public function setUp()
 
         $logger = $this->getMock('Psr\Log\LoggerInterface');
 
-        $this->pool = new Pool($container, '', '');
-        $this->pool->setAdminServiceIds(array('foo_admin', 'bar_admin'));
+        $this->pool = $this->getMockBuilder('Sonata\AdminBundle\Admin\Pool')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
```

Comment:
Please be at least consistent with your own code, pick one style and stick to it. I think we usually never put semi-colons on their own lines.